eleventy.config.js

```diff
@@ -1,9 +1,15 @@
+const eleventyAutoCacheBuster = require("eleventy-auto-cache-buster");
+
 module.exports = function(eleventyConfig) {
+  eleventyConfig.addPlugin(eleventyAutoCacheBuster);
 
   eleventyConfig.addPassthroughCopy('assets');
   eleventyConfig.setUseGitIgnore(false);
 
   return {
+    markdownTemplateEngine: "njk",
```

Comment:
the default for all these is liquid. It makes it so if you put some templating stuff in a file ending in .md or .html, it will just work! Which is nice but if you're expecting to use nunjucks syntax because that's what's in the other files, but it actually is getting parsed as liquid, you will occasionally have trouble there.